tests/python/contrib/test_tflite_runtime.py

```diff
@@ -14,47 +14,49 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""Configure pytest"""
+import importlib
 import pytest
-
-import tvm
-from tvm import te
 import numpy as np
+import tvm
 from tvm import rpc
 from tvm.contrib import utils, tflite_runtime
 
 
 def _create_tflite_model():
+    """Functions of creating a tflite model"""
     if not tvm.runtime.enabled("tflite"):
         print("skip because tflite runtime is not enabled...")
-        return
+        return None
     if not tvm.get_global_func("tvm.tflite_runtime.create", True):
         print("skip because tflite runtime is not enabled...")
-        return
+        return None
 
     try:
-        import tensorflow as tf
+        tf_lib = importlib.import_module("tensorflow")
```

Comment:
It's better to use `# pylint: disable=import-outside-toplevel` in this file, rather than importlib

Reply:
Got it. I will alter them.